package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "mozilla-leadership-network",
+  "name": "network",
```

Comment:
I guess...since it's not actually used it doesn't matter, but that's not a name that would be useful with a package manager. As it is I think it's a bit weird to just be a `mozilla/network` repo